docker-compose.yml

```
@@ -0,0 +1,29 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+    command: >
+      sh -c "./wait_for_db.sh && python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
```

it should look something like this:
```suggestion
      sh -c "
      python manage.py wait_for_db
      && python manage.py migrate 
      && python manage.py runserver 0.0.0.0:8000"
```